### PR TITLE
Feature  handle story deleted

### PR DESCRIPTION
**Pull request by MySolace**
Deal with deleted posts.

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
   "post_reply_title": "This is a reply to: {title}",
   "post_reply_show_original_post": "Show original post",
   "post_reply_show_parent_discussion": "Show parent discussion",
+  "post_eleted": "This post has been deleted",
```
Comment (piem) [inline src/locales/en.json]: did you mean `post_deleted`?

File: .circle/circle.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Node CircleCI 2.0 configuration file
+#
+#
 version: 2
```
Comment (piem) [inline .circle/circle.yml]: This change seems unrelated to the current PR. Any specific reason to include it here?

File: src/locales/en.json (hunk 1 above)
Comment (MySolace) [reply src/locales/en.json]: And this is why code reviews ... yes certainly, `post_deleted`.

File: .circle/circle.yml (hunk 2 above)
Comment (MySolace) [reply .circle/circle.yml]: This part wasn't intended to make it into the actual pull.
